Basestation_Software.Api/Entities/REDDatabase.cs

```diff
@@ -18,6 +20,16 @@ public REDDatabase(IConfiguration configuration)
     {
         // Assign member variables.
         Configuration = configuration;
+
+        // Attempt to form missing tables in data.db
+        try
+        {
+            (Database.GetService<IDatabaseCreator>() as RelationalDatabaseCreator).CreateTables();
+        }
+        catch
+        {
+            // CreateTables throws error if table already exists but we don't care
```

Comment:
Even though we don't care about the error it might be still a good idea to do something in this block. At least log something to the console or something.